website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,174 @@
+---
+layout: docs
+page_title: Automated entitlement utilization reporting
+description: >-
+  Learn what data HashiCorp collects to meter Enterprise entitlement utilization. Enable or disable reporting. Review sample payloads and logs.
+---
+
+# Automated entitlement utilization reporting
+
+Automated entitlement utilization reporting sends minimal product-entitlement
+metering data to HashiCorp without requiring you to manually collect and report
+them. It also lets you review your entitlement usage with the monitoring
+solution you already use (Splunk, Datadog, etc.) so you can optimize and manage
+your deployments. 
+
+Utilization reporting shares the minimum data required to validate entitlement
+utilization as contractually obligated. They consist of mostly computed metrics
+and will never contain PII or other sensitive information. Utilization reporting
+shares the data with HashiCorp using a secure, unidirectional HTTPS API, and
+makes an auditable record in the product logs each time it submits a report.
+
+To enable automated reporting you need to upgrade to a version that supports it,
+and open your network to outbound HTTPS traffic on port 443 from
```

Comment:
We use "to" elsewhere in the template (line 35) and I think it's more correct here.

```suggestion
and open your network to outbound HTTPS traffic on port 443 to
```